### ISSUE TITLE
FreeGeoIP deprecation

### ISSUE THREAD

**Issue by pedrojuk** [camelinaction/camelinaction2#126]
The Spring Boot examples in Chapter 7 try to return the current location as part of the Hello from Spring Boot and Camel message. The response now includes a deprecation message:
```json
{
"__deprecation_message__": "This API endpoint is deprecated and will stop working on July 1st, 2018. For more information please visit: https://github.com/apilayer/freegeoip#readme",
"ip": "nn.nn.nn.nn",
"country_code": "GB",
"country_name": "United Kingdom",
"region_code": "ENG",
"region_name": "England",
"city": "Balham",
"zip_code": "SW17",
"time_zone": "Europe/London",
"latitude": 51.4307,
"longitude": -0.1648,
"metro_code": 0
}
```
The readme at https://github.com/apilayer/freegeoip#readme says that there is a new API.  Perhaps the examples should be updated to use this?

**Comment by davsclaus** [comment]
There is a PR on ApacheCamel to update to a newer API that will be in Camel 2.23.

**Comment by davsclaus** [comment]
Okay this will be fixed by #128
